cmd/platform/deployment-template/list.go

```diff
@@ -55,5 +57,6 @@ func init() {
 	Command.AddCommand(listCmd)
 	listCmd.Flags().BoolP(showInstanceConfigurations, "", false, "Shows instance configurations - only visible when using the JSON output")
 	listCmd.Flags().String(stackVersion, "", "If present, it will cause the returned deployment templates to be adapted to return only the elements allowed in that version.")
-	listCmd.Flags().String(filter, "", "Optional key/value pair in the form of key:value that will act as a filter and exclude any templates that do not have a matching metadata item associated")
+	listCmd.Flags().String(filter, "", "Optional key/value pair in the form of key:value that will act as a filter and exclude any templates that do not have a matching metadata item associated.")
+	listCmd.Flags().String(format, "deployment", "If deployment is specified deployment_template is populated in the response, If cluster is specified cluster_template is populated in the response. (Defaults to deployment)")
```

Comment:
```suggestion
	listCmd.Flags().String(format, "deployment", "If a deployment is specified, the deployment_template is populated in the response. If cluster is specified, the cluster_template is populated in the response. (Defaults to deployment.)")
```

(We should not omit indefinite and definite articles, as they can help a global audience. The text should also be easier to parse with some added punctuation.)